<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calc</class>
 <widget class="QMainWindow" name="calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>460</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>61</width>
    <height>61</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Калькулятор польской записи чисел</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Никита/OneDrive/Desktop/Qt/image/calc.png</normaloff>C:/Users/Никита/OneDrive/Desktop/Qt/image/calc.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background-image: url(:/new/prefix1/image/calc.jpg);
}
QMessageBox
{
	
	background-color: rgb(74, 74, 74);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow
{
background-image: url(:/new/prefix1/image/calc.png);
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" rowspan="2" colspan="4">
     <widget class="QLabel" name="result_show">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 25 10pt &quot;Yu Gothic UI Light&quot;;
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid black;

background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pb9">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pb5">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pb6">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="mult">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pb3">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="equally">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}

</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="deg">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="clearAC">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
</string>
      </property>
      <property name="text">
       <string>⌫</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pb0">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="spaceb">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
      </property>
      <property name="text">
       <string>Пробел</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="div">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="minus">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pb7">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2" alignment="Qt::AlignBottom">
     <widget class="QPushButton" name="point">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="plus">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 0, 90);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pb8">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pb4">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pb2">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pb1">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="clearC">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>61</width>
        <height>61</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
